boards/arduino-atmega-common/include/gpio_params.h

```diff
@@ -34,7 +34,8 @@ static const  saul_gpio_params_t saul_gpio_params[] =
     {
         .name = "LED",
         .pin = LED0_PIN,
-        .mode = GPIO_OUT
+        .mode = GPIO_OUT,
```

Comment:
Does this mode field mean that when using saul, a GPIO pin config (in/out, pullups) ist hard-coded at compile time?

Reply:
I think so, then at runtime in `auto_init_gpio.c` the `mode` field is used to select in or out saul driver and is given to `gpio_init`.